README.md

```diff
@@ -1,236 +1,71 @@
-  ```sh
-  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-  ```
-* Redis
-
-  you can install Redis
-  ```sh
-  curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg
-  echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list
-  sudo apt-get update
-  sudo apt-get install redis
-  ```
-  or you can start a redis instance in a container.
-  ```sh
-  docker run --name test-redis -d -p 6379:6379 redis
-  ```
-  see [here](https://hub.docker.com/_/redis) for more info
-* yggdrasil
-
-  **note:** though not required to run fully functionally RMB, it needed if you are going to communicate with other nodes on our planetary network ).
-  see the [installation instructions](https://yggdrasil-network.github.io/installation-linux-deb.html)
-### Installation from source
-
-1. Register a twin at [tfchain](https://polkadot.js.org/apps/?rpc=wss://tfchain.grid.tf/ws#/accounts). for more info see [Grid3 Get Started](https://library.threefold.me/info/manual#/getstarted/manual__tfgrid3_getstarted) guide.
-2. make sure your yggdrasil service is up and that your ygg address was added to your twin either from [polkadot.js UI](https://polkadot.js.org/apps/?rpc=wss://tfchain.grid.tf/ws#/extrinsics) or from the [grid portal](https://portal.grid.tf/).
-3. Clone the repo.
-   ```sh
-   git clone https://github.com/threefoldtech/rmb-rs.git
-   ```
-3. Build and install the Daemon binary.
-   ```sh
-   cargo install --path .
-   ```
-
-in case of any build issues, things to try:
-- switch to the stable channel if you are on a nightly channel.
-
-  ```sh
-  rustup default stable
-  ```
-
-- rustup can be used to update the installed version to the latest release.
-
-  ```sh
-  rustup update
-  ```
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-
-
-<!-- USAGE EXAMPLES -->
-## Usage
-
-Example of using sr25519 key on dev network
-
-```sh
-rmb-rs --key-type sr25519 -s "wss://tfchain.dev.grid.tf:443" -m "<YOUR-MNEMONICS>"
+[![Rust](https://github.com/threefoldtech/rmb-rs/actions/workflows/rust.yaml/badge.svg)](https://github.com/threefoldtech/rmb-rs/actions/workflows/rust.yaml)
+
+# Reliable Message Bus
+Reliable message bus is a secure communication panel that allows `bots` to communicate together in a `chat` like way. It makes it very easy to host a service or a set of functions to be used by anyone, even if your service is running behind NAT.
+
+Out of the box RMB provides the following:
+- Grantee authenticity of the messages. You are always sure that the received message is from whoever is pretending to be
+- End to End encryption
+- Support for 3rd party hosted relays. Anyone can host a relay and people can use it safely since there is noway messages can be inspected while using e2e. That's similar to `home` servers by `matrix`
```

Comment:
```suggestion
- Support for 3rd party hosted relays. Anyone can host a relay and people can use it safely since there is no way messages can be inspected while using e2e. That's similar to `home` servers by `matrix`
```